Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridDataConnection.cs

```diff
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
+    // Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This is causing build.bat to fail. It should be undone.

Reply:
@RBrid This is weird as the line is still there in the file I committed and pushed

Reply:
@RBrid Sorry about that. Fixed